### ISSUE TITLE
"room_setpoint_temperature" is going crazy

### ISSUE THREAD

**Issue by mathiaslehmann** [erikkastelec/hass-WEM-Portal#29]
Hello, I noticed today that since the last update, the statistics of the "room_setpoint_temperature" entity are going crazy.  The value seems to be unreadable every now and then and is set to 0 at this moment.  Could it be related to the customization of the "heat_output" entity?  Thank you for the effort.

![Screenshot_20220424-230107_Home Assistant.jpg](https://user-images.githubusercontent.com/16978467/164996404-5cf8be8b-f9e0-4820-b76a-9b383ddd7f00.jpg)

**Comment by erikkastelec** [comment]
Will look into it.

**Comment by erikkastelec** [comment]
@mathiaslehmann is this still a problem and does it happen all the time? I am asking as I tried to reproduce the bug, but couldn't.

**Comment by mathiaslehmann** [comment]
@erikkastelec yes the problem is still there. It may be that if the value cannot be retrieved (connection problems, etc.), then the value is automatically set to 0. But I have to admit that my API scan interval is set to 100.

**Comment by mathiaslehmann** [comment]
From the feeling the behavior after this adjustment occurred like this. #27
